wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallDegradationViewController/CallDegradationController.swift

```diff
@@ -67,7 +68,7 @@ final class CallDegradationController: UIViewController {
                     continueDegradedCall ? self?.delegate?.continueDegradedCall() : self?.delegate?.cancelDegradedCall()
                 })
             }
-        case .none, .incoming:
+        case .none, .incoming, .terminating:
```

Comment:
So this controller only handles showing the alert for outgoing calls now?

Reply:
@johnxnguyen I have a second PR for proteus call to do, I'll remove the code if I see it's not needed